modules/search/js/search-widget-filters-admin.js

```diff
@@ -8,12 +8,60 @@
 	$( document ).ready( function() {
 		setListeners();
 
+		window.JetpackSearch = window.JetpackSearch || {};
+		window.JetpackSearch.addFilter = addFilter;
+
 		// Initialize Tracks
 		if ( 'undefined' !== typeof analytics && args.tracksUserData ) {
 			analytics.initialize( args.tracksUserData.userid, args.tracksUserData.username );
 		}
 	} );
 
+	// auto-generate title
+	function capitalize( str ) {
+		return str.charAt(0).toUpperCase() + str.slice(1);
+	}
+
+	function titleize( str ) {
+		var string_array = str.split(' ');
+		string_array = string_array.map(function(str) {
+		   return str.capitalize();
+		});
+
+		return string_array.join(' ');
+	}
+
+	function generateFilterTitlePlaceholder( container ) {
```

Comment:
This seems to currently be coming up with a user's name as the default for all filter types for some reason.

Reply:
I _think_ I fixed this - try again?